portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -20,13 +20,22 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
+import com.google.gson.Gson;
+
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  String[] ArrayList = {"name : Jamila", "Age : 20", "City : Adelaide"};
```

Comment:
variables should start with a lower case letter. Also ArrayList is a slightly confusing thing to name a variable, since ArrayList is also a type, but this variable is a string array not a string array list.

Reply:
changed it to array and will note that it should be always lowercase. I misunderstood the given problem, I thought they are asking me to create a varible string called arraylist, not a type of variable "arraylist" but I get it now